pages/bare_metal_cloud/dedicated_servers/rcw-changing-admin-password-on-windows/guide.en-gb.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "Reset Administrator password when using Rescue-Customer-Windows"
+excerpt: "Reset Administrator password when using Rescue-Customer-Windows"
+updated: 2024-05-21
+---
+
+## Objective
+
+This guide will help you to reset your `Administrator`'s password with the new __*rescue-customer-windows*__
+
+## Requirements
+
+- Microsoft Windows installed on your [dedicated server](/links/bare-metal/bare-metal)
+- At least 16 GB of RAM installed on the server
+- Access to the [OVHcloud Control Panel](/links/manager)
+
+> [!warning]
+>
+> This is not compatible with the `WinPE Rescue`.  
+> Use this [page](/pages/bare_metal_cloud/dedicated_servers/changing-admin-password-on-windows/guide.en-gb.md) if you are using the `WinPe Rescue`.
+>
+
+## Instructions
+
+### Step 1: Rebooting the server into rescue mode <a name="step1"></a>
+
+The system has to be started in rescue mode (WinRescue) before the admin password can be changed.
+
+For detailed instructions, please refer to the rescue mode guide.
+
+
+### Step 2: Clearing the current password <a name="step2"></a>
+
+Connect to your server in Remote Desktop using the credentials provided by email.
+
+Please note that the user name for the rescue is `Administrator`.
+
+If you server have software raid on your Windows disk, you have to import your Windows local disk to be able to access it : follow instructions of [section A](#sectionA)
```

Comment:
```suggestion
If you server uses software RAID for your Windows disk, you have to import your Windows local disk to be able to access it: follow the instructions of [section A](#sectionA)
```
Here, what we call "disk" is actually a "partition"? Or should it be plural because we refer to multiple physical disks?

Reply:
Better not enter technical complexity of windows software raid definitions of disk/partition/volume/plex, for the sake of customers' sanity. 
And "Windows disk" because if you have raid on non system disk it won't matter when resetting password.